kernel/build-kernel.sh

```diff
@@ -270,7 +271,13 @@ get_config_and_patches() {
 	if [ -z "${patches_path}" ]; then
 		info "Clone config and patches"
 		patches_path="${default_patches_dir}"
-		[ -d "${patches_path}" ] || git clone "https://${patches_repo}.git" "${patches_repo_dir}"
+		if [ ! -d "${patches_path}" ]; then
```

Comment:
Why should build kernel, from packaging, clone the packaging repo? I think we should assume that the caller will checkout the correct repo from which they build?

Reply:
I know this should not be required if you are on the correct tag while making the release, but this ensures that the correct patches are applied even if you are not.
I am in favor of removing this clone in the future, provided we first document this in our release process. (I have heard diff accounts of making a release when I started out.)